packages/block-library/src/image/edit.native.js

```diff
@@ -0,0 +1,10 @@
+/** @format */
+
+import { View, Text } from 'react-native';
```

Comment:
Well, I noticed one small thing now though: let's add a section header like this one: https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/heading/edit.native.js#L6-L9